packages/notifi-graphql/lib/gql/fragments/DiscordTargetFragment.gql.ts

```diff
@@ -10,5 +10,6 @@ export const DiscordTargetFragment = gql`
     name
     userStatus
     verificationLink
+    discordServerInviteLink
```

Comment:
we need to add it to `notifi-axios-adapter/lib/fragments/discordTargetFragment.ts` as well